MovieDB/Extension/UIViewController+.swift

```diff
@@ -19,37 +19,44 @@ extension UIViewController {
         self.present(alert, animated: true, completion: nil)
     }
     
-    func handleIndicator(type: IndicatorType) {
+    func handleIndicator(_ type: IndicatorType) {
+         switch type {
+         case .show:
+             configActivityIndicator(.show)
+             UIViewController.activityIndicator.startAnimating()
+             view.isUserInteractionEnabled = false
+         case .hide:
+             UIViewController.activityIndicator.stopAnimating()
+             view.isUserInteractionEnabled = true
+             configActivityIndicator(.hide)
+         }
```

Comment:
dùng toán tử 3 ngôi được k e

Reply:
@phonglh-2668 Dạ a ơi cho luồng Main Thread ở 2 case khác nhau nên không dùng toán tử 3 ngôi được ạ. Như ở show thì phải config xong rồi mới show lên được, còn ở case hide thì phải stop rồi mới hide rồi mới config gỡ subview ạ

Reply:
@trunglnt-3212 e thử `type == .show ? UIViewController.activityIndicator.startAnimating() :  UIViewController.activityIndicator.stopAnimating()` chưa